Documentation/CrossPlatform/OculusQuestMRTK.md

```diff
@@ -20,26 +25,80 @@ project which uses the Legacy XR pipeline.
 
 1. Ensure that [developer mode](https://developer.oculus.com/documentation/native/android/mobile-device-setup/) is enabled on your device. Installing the Oculus ADB Drivers is optional.
 
-1. Adding the Oculus XRSDK Data Provider
+## Setting up the scene
     - Create a new Unity scene or open a pre-existing scene like HandInteractionExamples
+    
     - Add MRTK to the scene by navigating to **Mixed Reality Toolkit** > **Add to Scene and Configure**
```

Comment:
This section is showing up like this in the rich diff
![image](https://user-images.githubusercontent.com/53493796/90679621-45b67f00-e215-11ea-9864-23500a5f0cb7.png)

I think something like this is the fix:
```suggestion
## Setting up the scene
1. Create a new Unity scene or open a pre-existing scene like HandInteractionExamples
1. Add MRTK to the scene by navigating to **Mixed Reality Toolkit** > **Add to Scene and Configure**
```